src/sentry/tagstore/types.py

```diff
@@ -110,6 +118,16 @@ def serialize(self, obj, attrs, user, **kwargs):
         return output
 
 
+class TagValueSerializerResponse(TypedDict):
+    key: str
+    name: str
+    value: str
+    count: int
+    lastSeen: str
+    firstSeen: str
+    query: str | None
```

Comment:
if this is optional in the response (it might not contain this field), you might need to use https://develop.sentry.dev/application/api/public/#success-responses

take a look at the section on 
> In order to indicate optional fields that may or may not be set in the response, you must use [totality](https://peps.python.org/pep-0589/#totality) and separate your TypedDict into two classes.